great_expectations/data_context/store/profiler_store.py

```diff
@@ -59,3 +59,14 @@ def serialization_self_check(self, pretty_print: bool) -> None:
             print(
                 f"\tTest key and value successfully removed from Profiler store: {test_value}\n"
             )
+
+    def ge_cloud_response_json_to_object_dict(self, response_json: dict) -> dict:
+        """
+        This method takes full json response from GE cloud and outputs a dict appropriate for
+        deserialization into a GE object
+        """
+        ge_cloud_profiler_id = response_json["data"]["id"]
+        profiler_config_dict = response_json["data"]["attributes"]["profiler_config"]
```

Comment:
```suggestion
        profiler_config_dict = response_json["data"]["attributes"]["profiler"]
```

to correspond with previous comment

Reply:
done!